### PR TITLE
feat: support multiple avatars for `DtRecipeContactInfo` Component

### PR DESCRIPTION
**Pull request by SheldonDialpad**
# PR Title

Support multiple avatars for `DtRecipeContactInfo` Component

## :hammer_and_wrench: Type Of Change

- [ ] Fix
- [x] Feature
- [ ] Refactoring
- [ ] Documentation

## :pencil: Checklist

- [x] I have reviewed my changes
- [x] I have added all relevant documentation
- [x] I have validated components with a screen reader
- [x] I have considered the performance impact of my change
- [x] I have checked that my change did not significantly increase bundle size

## :crystal_ball: Next Steps

Update Application UI with multiple avatars feature.

## :camera: Screenshots / GIFs

Screen Shot 2023-03-31 at 12 36 47 AM
Screen Shot 2023-03-31 at 12 37 57 AM

## :link: Sources

Jira: [DP-66880](https://dialpad.atlassian.net/browse/DP-66880)

[DP-66880]: https://dialpad.atlassian.net/browse/DP-66880?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by bhawkins-dialpad** [comment]
@SheldonDialpad The margin between the avatars and the text only needs to be 4 here since there is an extra 4px around the avatars for the white border. 
image

The vertical spacing on the text is a little inconsistent in some places
image

**Comment by braddialpad** [comment]
You can update to dialtone-icons 1.3.2 and the new icon should be there.

File: recipes/list_items/contact_info/contact_info.vue [hunk 1]
```diff
@@ -133,6 +172,19 @@ export default {
...
       default: null,
     },
+
+    /**
+     * Showing multiple avatars in contact info.
+     * The props of array items are:
+     * `src` - avatar image url
+     * `initials` - Initial letters to display in avatar if `src` is empty.
+     * `text` - cover text that showing over avatar
+     * `icon` - cover icon that showing over avatar
```
Comment (braddialpad) [inline recipes/list_items/contact_info/contact_info.vue]: State which of these are required.

Comment (braddialpad) [reply recipes/list_items/contact_info/contact_info.vue]: Screenshot 2023-04-04 at 9 58 01 AM

Unfortunately these line breaks don't translate into storybook very well, you can add `<br>` at the end of each line to get them to work.

Comment (braddialpad) [inline recipes/list_items/contact_info/contact_info.vue]: "text that overlays the avatar"

"icon that overlays the avatar"

Comment (braddialpad) [reply recipes/list_items/contact_info/contact_info.vue]: Does it make sense to have text and icon on each individual avatar? or should it always be on the one in front in which case they could be their own props.

**Review by braddialpad — COMMENTED** [review]
Sorry for the delay on this review, thanks for the contribution!

File: recipes/list_items/contact_info/contact_info.vue (hunk 1 above)
Comment (SheldonDialpad) [reply recipes/list_items/contact_info/contact_info.vue]: updated

Comment (SheldonDialpad) [reply recipes/list_items/contact_info/contact_info.vue]: I think if we want to support showing some info for each user (for example, admin listens in group calls, multiple admins listen in agent's call, etc) in the future, we don't need change the `DtRecipeContactInfo`.
multiple-avatars
(admin listens in group calls)
Screen Shot 2023-04-05 at 7 16 06 AM
(multiple admins listen in agent's call)

Comment (braddialpad) [reply recipes/list_items/contact_info/contact_info.vue]: okay yeah if that's a valid case, definitely leave it as is.

**Review by braddialpad — APPROVED** [review]
Looks good by me

**Comment by braddialpad** [comment]
@bhawkins-dialpad Did you want to have a final check?

**Comment by bhawkins-dialpad** [comment]
@SheldonDialpad @braddialpad This looks better thank you!

**Pull request merged**